plugins/woocommerce-admin/client/guided-tours/wc-addons-tour/index.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { TourKit, TourKitTypes } from '@woocommerce/components';
+import qs from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import { waitUntilElementTopNotChange } from '../add-product-tour/utils';
```

Comment:
If `../add-product-tour/utils.js` is used by multiple tours, then I would consider moving `utils.js` file one level up to `plugins/woocommerce-admin/client/guided-tours`. Then it'll be more obvious that those utils are not specific to `add-product-tour` but can rather be reused by different tours.